### PR TITLE
Update Xion testnet IBC, versions and endpoints

### PR DESCRIPTION
**Pull request by froch**
- Add INJ IBC
- Update versions list
- Update current version
- Remove deprecated endpoints

### CODE REVIEW

File: testnets/xiontestnet/assetlist.json [hunk 1]
```diff
@@ -181,6 +174,52 @@
...
+        },
+        {
+          "denom": "INJ",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/0166AE2224341A3F70943E315DAC6EDF012A638D0F9358794FF7525BA1DFC191",
+      "name": "Injective INJ token",
+      "display": "inj",
```
Comment (JeremyParish69) [inline testnets/xiontestnet/assetlist.json:195]: display denom would be "INJ" all uppercase, not 'inj'

File: testnets/xiontestnet/assetlist.json [hunk 2]
```diff
@@ -181,6 +174,52 @@
...
+      "symbol": "INJ",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injectivetestnet",
+            "base_denom": "inj",
+            "channel_id": "channel-324"
```
Comment (JeremyParish69) [inline testnets/xiontestnet/assetlist.json:203]: Please register this IBC connection

Comment (froch) [reply testnets/xiontestnet/assetlist.json:203]: make sense, thank you. updated to `INJ`

File: testnets/xiontestnet/assetlist.json (hunk 1 above)
Comment (froch) [reply testnets/xiontestnet/assetlist.json]: make sense, thank you. updated to `INJ`

File: testnets/xiontestnet/assetlist.json (hunk 2 above)
Comment (froch) [reply testnets/xiontestnet/assetlist.json:203]: registered to `_IBC`, mirroring live connection. thank you

```
# Chain: injective-888
  - Client: 07-tendermint-228
    * Connection: connection-211
      | State: OPEN
      | Counterparty state: OPEN
      + Channel: channel-324
        | Port: transfer
        | State: OPEN
        | Counterparty: channel-215
# Chain: xion-testnet-1
  - Client: 07-tendermint-105
    * Connection: connection-43
      | State: OPEN
      | Counterparty state: OPEN
      + Channel: channel-215
        | Port: transfer
        | State: OPEN
        | Counterparty: channel-324
```

**Pull request merged**